platform-api/che-core-api-user/src/main/java/org/eclipse/che/api/user/shared/dto/UserInRoleDescriptor.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.user.shared.dto;
+
+
+import com.wordnik.swagger.annotations.ApiModelProperty;
+
+import org.eclipse.che.dto.shared.DTO;
+
+/**
+ * Defines if the current user is or not in a current role
+ * @author Florent Benoit
+ */
+@DTO
+public interface UserInRoleDescriptor {
+
+    @ApiModelProperty(value = "Is in Role")
+    boolean getIsInRole();
```

Comment:
DTO engine [handles it](https://github.com/codenvy/che-core/blob/master/platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/shared/dto/WorkspaceDescriptor.java#L41-L46) as well